webpack.config.js

```diff
@@ -0,0 +1,12 @@
+const pkg = require('./package.json');
+
+module.exports = {
+    entry: {
+        'injector.lite': './src/entries/injector.lite.js',
```

Comment:
I have already changed it over to looking for injector.lite.js in the header, but the problem is that injector.lite.js is webpacked and is webpacked again in the cnn-header which leads to an error. So we might not be able to webpack injector